frontend/src/router/index.js

```diff
@@ -10,13 +9,17 @@ import HistoryPage from '@/pages/HistoryPage';
 import SeachPlacePage from  '@/pages/SeachPlacePage';
 import store from '../store/index';
 import middlewares from './middlewares';
+import SignUp from '@/components/auth/SignUp';
+import UserListAdd from '@/components/user-list/UserListAdd';
```

Comment:
Create add list page component container in `/pages/UserListAddPage.vue`. Further, you can create `AddListForm.vue` into /components/userList and place into UserListAddPage.vue